src/vm/assemblynative.cpp

```diff
@@ -1296,40 +1296,37 @@ INT_PTR QCALLTYPE AssemblyNative::GetLoadContextForAssembly(QCall::AssemblyHandl
    
     // Platform assemblies are semantically bound against the "Default" binder which could be the TPA Binder or
     // the overridden binder. In either case, the reference to the same will be returned when this QCall returns.
```

Comment:
You can delete "or the overriden binder" part. It does not exist anymore (you have deleted last bit of it in the last PR).